.github/workflows/deploy_production.yml

```diff
@@ -0,0 +1,38 @@
+name: Deploy to Production
+
+on:
+  push:
+    branches:
+      - main
+  workflow_dispatch:
+
+jobs:
+  get_date:
+    runs-on: ubuntu-latest
+    outputs:
+      date: ${{ steps.date_getter.outputs.date }}
+    steps:
+      - id: date_getter
+        run: echo "date=$(date '+%A %W %Y %X')" >> $GITHUB_OUTPUT
+
+  build_and_push_image:
+      name: Build and Push Image
+      uses: zooniverse/ci-cd/.github/workflows/build_and_push_image.yaml@main
+      needs: get_date
+      with:
+        repo_name: subject-set-search-api
+        commit_id: ${{ github.sha }}
+        latest: true
+        build_args: |
+          BUILD_DATE: ${{ needs.get_date.outputs.date }}
```

Comment:
`BUILD_DATE` is printed as an empty string when this runs, which means that new data isn't read from the Panoptes API.
https://github.com/zooniverse/subject-set-search-api/actions/runs/4803174012/jobs/8547411836#step:7:151

Reply:
No wait, `BUILD_DATE` isn't printed for a successful build either. I'm not sure why this branch is ignoring it in the image.
https://github.com/zooniverse/subject-set-search-api/actions/runs/4806214177/jobs/8553461972#step:4:314